p2p/discv5/net_test.go

```diff
@@ -336,26 +336,26 @@ func (*preminedTestnet) localAddr() *net.UDPAddr {
 
 // mine generates a testnet struct literal with nodes at
 // various distances to the given target.
-func (n *preminedTestnet) mine(target NodeID) {
-	n.target = target
-	n.targetSha = crypto.Keccak256Hash(n.target[:])
+func (tn *preminedTestnet) mine(target NodeID) {
```

Comment:
We could use `net` here.

Reply:
From what I can tell, it looks like ```net``` is being reserved by the imported 'net' package net.ParseIP("10.0.2.99"), so I've changed the variable name back to tn.